app/views/content/life-as-a-teacher/explore-subjects/modern-languages/_article.html.erb

```diff
@@ -0,0 +1,142 @@
+        <li>more holiday days than people in many other professions</li>
+        <li>opportunities for progression</li>
+      </ul>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "There’s a lot of sharing of stories and the kids can be so funny, they really crack me up. Plus, I enjoy the holidays. I get to travel still and can keep up with my languages.",
+    name: "Jacqui, languages teacher",
+    large: true
+  ) %>
+  </section>
+
+<section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Learning a new language will give pupils cultural awareness and enhanced communication skills that can lead to exciting careers.</p>
+      <p>Popular careers range from flight attendants and ski instructors to interpreters and tour guides. Pupils may also have advantages in careers such as journalism, public relations or finance.</p>
+      <p>Your pupils will learn more from you that you might think. As their teacher, you will guide your pupils though the skills they need to show their understanding of modern languages. You will teach them valuable life skills such as enhanced listening, cultural awareness and communication.</p>
+  </section>
+
+    <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "Teaching languages is so much more than teaching grammar and vocabulary. You get to introduce students to new cultures and a different way of doing things. In an increasingly globalised world, you’re providing kids with an incredibly valuable skill that they’ll cherish for their entire lives.",
+    name: "Tom, languages teacher",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-languages-progammes-of-study/national-curriculum-in-england-languages-progammes-of-study#key-stage-3-modern-foreign-language">national curriculum for modern foreign languages</a>.</p>
+    <p>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</p>
+    <ul>
+      <li>identifying and using present, past, and future tenses</li>
+      <li>using a variety of key grammatical structures and patterns</li>
+      <li>developing and using a wide-ranging and deepening vocabulary</li>
+      <li>using accurate grammar, spelling and punctuation</li>
+      <li>having conversations and making use of important social conventions such as formal modes of address</li>
+      <li>expressing and developing ideas clearly and with increasing accuracy</li>
+      <li>speaking coherently and confidently</li>
+      <li>reading literary texts in the language, so pupils can expand their understanding of the language and culture</li>
+      <li>writing creatively to express their own ideas and opinions</li>
+    </ul>
+  </section>
```

Comment:
Need to remove the section here - this is a hangover from the old design with horizontal bars and is creating a big gap between paragraphs